### DISCUSSION TITLE
Calculating Distance Between Loco Modules

### DISCUSSION THREAD

**Discussion by Deeshant2234** [bitcraze/discussions#507] [Q&A]
Actually I want to use UWB capability of the Loco. Can you please help me that how we can find the distance between two loco nodes using UWB?

**Comment by krichardsson** [comment]
I think I need some more information of your use case and your setup. Do you want to acquire the distance from the USB port on one of the nodes or from a Crazyflie?

**Reply by duembgen** [reply]
I might have a related question -- it says in a few places that the Loco Positioning Node can be used standalone -- is this functionality supported and documented in more detail anywhere? I am mostly curious about the best way to log data from the node (once configured as a tag). Logging through USB for instance would be ideal for us.

**Reply by krichardsson** [reply]
@duembgen This is a very good topic! Could you please start a new discussion for it as it is a bit too far away from the title
